app/init.php

```diff
@@ -196,7 +199,55 @@ function($value, Document $document, Database $database) {
         return $database
             ->find('indexes', [
                 new Query('collectionId', Query::TYPE_EQUAL, [$document->getId()])
-            ], 100, 0, []);
+            ], 5000, 0, []);
+    }
+);
+
+Database::addFilter('subQueryPlatforms',
+    function($value) {
+        return null;
+    },
+    function($value, Document $document, Database $database) {
+        return $database
+            ->find('platforms', [
+                new Query('projectId', Query::TYPE_EQUAL, [$document->getId()])
+            ], 5000, 0, []);
```

Comment:
@eldadfux what's an appropriate default limit for these queries? Is `APP_LIMIT_COUNT` seems too high for me.

Reply:
`param('limit', 25, new Range(0, 100),`
Put 25 and 100 into variables, use everywhere..
Then use 100 as a limit in all subqueries.

If you need more than 100, create API endpoint for this sub-collection.